spec/models/topic_spec.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_topics_on_name  (name) UNIQUE
+#
+require 'rails_helper'
```

Comment:
you can define a `.rspec` file where configure that all files require the helper file, so, you avoid adding this line everywhere.

Reply:
The downside of doing that is that sometimes you don't need to load the `rails_helper ` and it will slow down your tests. Policies and presenters are cases of that. I agree with adding it to `. rspec` for this case though.